django/db/backends/sqlite3/base.py

```diff
@@ -324,11 +324,17 @@ class SQLiteCursorWrapper(Database.Cursor):
     def execute(self, query, params=None):
         if params is None:
             return Database.Cursor.execute(self, query)
+        if isinstance(params, (tuple)):
+            if params:
+                if isinstance(params[0], (int, long)):
```

Comment:
There is no good solution to this,or We could explicitly store them as a string (i.e. call str before INSERT) and then cast back to int() on SELECT